oauthenticator/cilogon.py

```diff
@@ -59,6 +59,9 @@ def authorize_redirect(self, *args, **kwargs):
             extra_params["selected_idp"] = self.authenticator.idp
         if self.authenticator.skin:
             extra_params["skin"] = self.authenticator.skin
+        if self.authenticator.username_key:
```

Comment:
username_key is undefined. Do you mean for the `claim` trait below to be called `username_key`? Or are `claim` and `username_key` different concepts? It seems that the key from which we will retrieve the username and the claim requested via the oauth flow aren't the same thing.

Reply:
They are the same thing, in my mind. I was inclined to call it claim because that is what it is internally in cilogon, but I interpreted https://github.com/jupyterhub/oauthenticator/issues/133#issuecomment-335754031 as "let's call it `username_key`, so that users who do not know anything about cilogon will be less confused when needing to set that in `jupytherhub_config.py`". From what you are saying now, I think I probably misinterpreted that though

Reply:
Thanks, that makes the `claim` word make sense. Since I want it to be clear what this means - that it's specifically the claim to produce the username, let's call it `username_claim`.